lib/portage/exception.py

```diff
@@ -216,12 +216,14 @@ def __str__(self):
         if not isinstance(eapi, str):
             eapi = str(eapi)
         eapi = eapi.lstrip("-")
-        msg = _(
-            "Unable to do any operations on '%(cpv)s', since "
-            "its EAPI is higher than this portage version's. Please upgrade"
-            " to a portage version that supports EAPI '%(eapi)s'."
-        ) % {"cpv": self.cpv, "eapi": eapi}
-        return _unicode_decode(msg, encoding=_encodings["content"], errors="replace")
+        msg = " ".join(
+            (
+                f"Unable to do any operations on '{self.cpv}', since its EAPI is",
+                "higher than this portage version's. Please upgrade to a portage",
+                f"version that supports EAPI '{eapi}'.",
+            )
```

Comment:
I don't think we need the `join` here:

````python
>>> f"a" "b" f"c"
'abc'
````

I guess just continue using `_(...)` for the moment?